### ISSUE TITLE
Include file in validation output?

### ISSUE THREAD

**Issue by Peter-Optiway** [sasstools/sass-lint#1155]
We have a build.scss file that includes a lot of other files, but when i get validation errors it just says the line and column is there any way to include the file the issue occurred in?

```bash
client\assets\styles\build.scss
  10:5   warning  Class should be nested within its parent Type-selector                           force-element-nesting
  10:16  warning  Qualifying elements are not allowed for class selectors                          no-qualifying-elements
  17:5   warning  Class should be nested within its parent Class                                   force-element-nesting
  22:21  warning  Color 'lightgray' should be written in its hexadecimal form #d3d3d3              no-color-keywords
  22:21  warning  Color literals such as 'lightgray' should only be used in variable declarations  no-color-literals
  23:16  warning  Color 'white' should be written in its hexadecimal form #ffffff                  no-color-keywords
  23:16  warning  Color literals such as 'white' should only be used in variable declarations      no-color-literals
```

build.scss
```scss
@import 'menu';
```

if the error is in my `menu.scss` it is still listed under `client\assets\styles\build.scss`
